lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -23,7 +23,11 @@
   name:
     description:
       - name of the database to add or remove
-      - name=all May only be provided if I(state) is C(dump) or C(import).
+      - name=all May only be provided if I(state) is C(dump) or C(import)
```

Comment:
```suggestion
      - I(name=all) May only be provided if I(state) is C(dump) or C(import)
```
etc
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block
> I() for option names. For example: Required if I(state=present).

Reply:
Fixed